packages/synapse-react-client/src/components/RecentPublicationsGrid/RecentPublicationsGrid.tsx

```diff
@@ -0,0 +1,176 @@
+import React from 'react'
+import {
+  Grid,
+  Card,
+  CardContent,
+  Typography,
+  Button,
+  Box,
+  useMediaQuery,
+  useTheme,
+} from '@mui/material'
+import { SynapseConstants, SynapseUtilityFunctions } from '../../utils'
+import { QueryBundleRequest } from '@sage-bionetworks/synapse-types'
+import { getFieldIndex } from '../../utils/functions/queryUtils'
+import useGetQueryResultBundle from '../../synapse-queries/entity/useGetQueryResultBundle'
+import dayjs from 'dayjs'
+import { formatDate } from '../../utils/functions/DateFormatter'
+
+export type RecentPublicationsGridProps = {
+  sql: string
+  buttonLink?: string
+  buttonLinkText?: string
+  summaryText?: string
+}
+
+function RecentPublicationsGrid(props: RecentPublicationsGridProps) {
+  const { sql, buttonLink, buttonLinkText, summaryText } = props
+  const theme = useTheme()
+  const isSmallScreen = useMediaQuery(theme.breakpoints.down('sm'))
+  const isMediumScreen = useMediaQuery(theme.breakpoints.between('sm', 'md'))
+  const entityId = SynapseUtilityFunctions.parseEntityIdFromSqlStatement(sql)
+  const queryBundleRequest: QueryBundleRequest = {
+    partMask:
+      SynapseConstants.BUNDLE_MASK_QUERY_SELECT_COLUMNS |
+      SynapseConstants.BUNDLE_MASK_QUERY_RESULTS,
+    concreteType: 'org.sagebionetworks.repo.model.table.QueryBundleRequest',
+    entityId,
+    query: {
+      sql,
+    },
+  }
+  const { data: queryResultBundle } =
+    useGetQueryResultBundle(queryBundleRequest)
+
+  const dataRows = queryResultBundle?.queryResult!.queryResults.rows ?? []
+
+  enum ExpectedColumns {
+    TAG = 'Tag',
+    JOURNAL = 'Journal',
+    TITLE = 'Title',
+    PUBLICATION_DATE = 'publicationDate',
+  }
+
+  const tagColIndex = getFieldIndex(ExpectedColumns.TAG, queryResultBundle)
+  const journalColIndex = getFieldIndex(
+    ExpectedColumns.JOURNAL,
+    queryResultBundle,
+  )
+  const titleColIndex = getFieldIndex(ExpectedColumns.TITLE, queryResultBundle)
+  const publicationDateColIndex = getFieldIndex(
+    ExpectedColumns.PUBLICATION_DATE,
+    queryResultBundle,
+  )
+
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        flexDirection: isSmallScreen || isMediumScreen ? 'column' : 'row',
+        gap: isSmallScreen || isMediumScreen ? '38px' : '80px',
+      }}
+    >
+      <Grid
+        container
+        spacing={2}
+        sx={{
+          order: isSmallScreen || isMediumScreen ? 1 : 0,
+          flex: isSmallScreen || isMediumScreen ? '0 0 100%' : '3',
+        }}
+      >
+        {dataRows.map(pub => (
+          <Grid item xs={12} sm={6} md={4} key={pub.rowId}>
+            <Card sx={{ height: '100%', boxShadow: 'none', border: 'none' }}>
```

Comment:
Is there a reason you use the `Card` component here but remove (what looks like) all of the styling that makes it a 'card'? I think `Box`/`div` would be fine unless there's a style/accessibility improvement I'm unaware of that you get by using `Card`

Reply:
I had Card there mainly for semantics! With the way I was using it here, I don't think there should really be a difference between using Card or Box / div so I went ahead and replaced this with Box to keep it more simple.